cli/popper/commands/cmd_add.py

```diff
@@ -0,0 +1,76 @@
+import click
+import os
+import hcl
+import shutil
+import popper.utils as pu
+import popper.scm as scm
+from popper.cli import pass_context
+from distutils.dir_util import copy_tree
+
+
+@click.command('add', short_help='Import workflow from remote repo.')
+@click.argument('path', required=True)
+@click.option(
+    '--branch',
+    help='Specifies the branch to download from.',
+    required=False,
+    default='master'
+)
+@pass_context
+def cli(ctx, path, branch):
+    """Imports a workflow from a remote project to the current project
+    directory. For now it can import workflows only from repos hosted at
+    github.com
+    """
+    project_root = scm.get_root_folder()
+    if not pu.is_popperized(project_root):
+        pu.fail('Repository has not been popperized.')
+        return
+
+    parts = path.split('/')
+    parts = list(filter(None, parts))
+    if len(parts) < 2:
+        pu.fail('Remote url format should be  <repo>[/folder[/wf.workflow]]')
```

Comment:
we could extend the format to be:

```
<url>/<repo>/<folder>/wf.workflow
```

we might be able to refactor the logic that is used [here](https://github.com/systemslab/popper/blob/1d4fbe5/cli/popper/gha.py#L199-L.235) so that we can reuse it for `popper add` as well.